microhapdb/tests/test_cli.py

```diff
@@ -129,6 +129,7 @@ def test_main_marker_query(capsys):
   mh19KK-056 MHDBM-d6ff8635    GRCh38 chr19                                                4852124,4852324 2.4391 0.0773  0.1760                         ALFRED
   mh19CP-007 MHDBM-49dbcc57    GRCh38 chr19                                     14310739,14310772,14310780 3.0813 0.0466  0.0776                         ALFRED
 mh19USC-19pB MHDBM-76427274    GRCh38 chr19                   16040864,16040894,16040899,16040921,16040929 3.5107 0.1647  0.0731   10.1016/j.fsigen.2019.102213
+ mh19ZHA-006 MHDBM-5f597339    GRCh38 chr19          20579862,20579863,20579879,20579892,20579916,20579924 3.8739 0.1681 -0.0032   10.1016/j.fsigen.2020.102255
```

Comment:
`mh19ZHA-006` is a new marker that we added, and it resides on the 19th chromosome. Updated to be included in the test.